### DISCUSSION TITLE
notepad++ support?

### DISCUSSION THREAD

**Discussion by steve02081504** [community/community#19408] [Copilot Conversations]
I often use notepad++ more often than launching a complicated vs or vs code
So I'm wondering why copilot doesn't support notepad++?

**Comment by D7EAD** [comment]
Hi!

If I remember correctly, Copilot currently supports only Visual Studio Code, Neovim, and IntelliJ-based IDEs like JetBrains IntelliJ IDEA, Android Studio or PyCharm--as well as VS 2022.

On a sidenote, what, in your opinion, leads you away from using a code editor like VS Code? I highly suggest taking another go at it, as it is incredibly powerful and supports Copilot!

**Reply by steve02081504** [reply]
Because I've written some syntax highlighting files for niche languages on notepad++, but I don't have the energy to write another one for vs or vs code at the moment😂, especially since the documentation for the language support plugin for vs code is a bit complicated and confusing for me
On the other hand, notepad++ runs on little memory and is fast, so I'm used to using it.✨

**Reply by D7EAD** [reply]
Fair enough! Hopefully, in the future, we get to see Copilot support for Notepad++!

**Comment by Lu-233** [comment]
It's not a reliable editor anymore。 https://twitter.com/Notepad_plus/status/1618276938342359042
